developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,107 @@
 title: "Accounts"
 id: "basics-accounts"
 ---
-An account represents a resource on the Aptos Blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources.
 
-# Introduction
+# Accounts
 
-The state of each account comprises both code and data:
+An account on the Aptos Blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes on access control and scarcity in its representation of an asset.  
 
-- **Code**: Move modules contain code (type and procedure declarations), but they do not contain data. The module's procedures encode the rules for updating the blockchain's global state.
-- **Data**: Move resources contain data but no code. Every resource value has a type that is declared in a module published in the blockchain's distributed database.
+Each account on the Aptos Blockchain is identified by a 32-byte account address. An account can store data and it stores this data in resources. The initial resource is the account data itself (authentication key and sequence number). Additional resources like the currency or NFTs are added after creating the account. 
 
-An account may contain an arbitrary number of Move resources and Move modules.
+## Creating an account
 
-## Initial Account Setup
+When a new account is requested by the user, the [Aptos SDK](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) will perform a series of cryptographic steps to create the account. Below is a conceptual view of these steps:
 
-An Aptos account is referenced by an account address, which is a 32-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the 32-byte of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
+- Start by generating a new private key, public key pair.
+- From the user get the preferred signature scheme for the account: If the account should use a single signature or if it should require multiple signatures for signing a transaction. 
+- Combine the public key with the user's signature scheme to generate a 32-byte authentication key. 
+- Initialize the sequence number to 0. The sequence number represents the next transaction sequence number, to prevent replay attacks of transactions. Both the authentication key and the sequence number are stored in the account as an initial account resource. 
```

Comment:
"account sequence number" perhaps.

Reply:
and a sequence number beginning at 0 ... the Sequence number is used by transactions to prevent replay attacks?